src/Microsoft.AspNetCore.Hosting/WebHostBuilder.cs

```diff
@@ -111,12 +109,7 @@ public IWebHostBuilder ConfigureServices(Action<IServiceCollection> configureSer
         /// <returns>The <see cref="IWebHostBuilder"/>.</returns>
         public IWebHostBuilder ConfigureServices(Action<WebHostBuilderContext, IServiceCollection> configureServices)
         {
-            if (configureServices == null)
```

Comment:
What happens if you pass null?

Reply:
The reason you'd want to check for null in the current version of the code is so you don't have to check for it later when enumerating the list for invoking the delegates when a null entry would throw a NullReferenceException.
But Action<T> is a MulticastDelgate and as such it already embeds the functionality of invoking multiple delegates. It also takes care of null, so you only have to check for null on the "head" of the mulicast delegate.

https://docs.microsoft.com/en-us/dotnet/api/system.delegate.combine?view=netcore-2.1#System_Delegate_Combine_System_Delegate_System_Delegate_